tools/smtradfile.c

```diff
@@ -40,6 +40,9 @@
 #include "msg.h"
 #include "module-template.h"
 #include "unicode-helper.h"
```

Comment:
Please do these types of defines once and for all in the relevant headers where the base data type is defined. For msg_t, this is ./runtime/typedefs.c - this is the file where it usually happens.